### PR TITLE
[Feature][Connector V2] expose configurable options in Kudu

### PR DESCRIPTION
**Pull request by Carl-Zhou-CN**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)

### CODE REVIEW

**Comment by Carl-Zhou-CN** [comment]
@Hisoka-X  @EricJoy2048 Please help to review,thanks

**Comment by Carl-Zhou-CN** [comment]
link #3337

**Review by Hisoka-X — APPROVED** [review]
LGTM

File: seatunnel-connectors-v2/connector-kudu/src/main/java/org/apache/seatunnel/connectors/seatunnel/kudu/config/KuduSinkConfig.java [hunk 1]
```diff
@@ -26,11 +29,25 @@
...
+    public static final Option<String> KUDU_MASTER =
+        Options.key("kudu_master")
+            .stringType()
+            .noDefaultValue()
+            .withDescription("kudu master address");
+
+    public static final Option<SaveMode> KUDU_SAVE_MODE =
+        Options.key("save_mode")
+            .enumType(SaveMode.class)
+            .noDefaultValue()
```
Comment (hailin0) [inline seatunnel-connectors-v2/connector-kudu/src/main/java/org/apache/seatunnel/connectors/seatunnel/kudu/config/KuduSinkConfig.java:41]: set default value to `APPEND`?

Comment (Carl-Zhou-CN) [reply seatunnel-connectors-v2/connector-kudu/src/main/java/org/apache/seatunnel/connectors/seatunnel/kudu/config/KuduSinkConfig.java:41]: At present, this parameter is defined as mandatory. If necessary, I can change it to non mandatory and add the default value

**Review by EricJoy2048 — APPROVED** [review]

**Pull request merged**